### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by ethantakla** [zju3dv/EfficientLoFTR#17]
Hello, I just ported over your code only to realize at the end there is no loss function. Ending up having to implement it myself in the mean time based on the paper, wondering when this will be available so I can use the official (and most definitely) more accurate version from you and the team?

Thanks and best,

Ethan

**Comment by wyf2020** [comment]
Thank you for your interest in our work! Our training code is currently being organized and is expected to be open-sourced this month.

**Comment by ethantakla** [comment]
Thank you for the clarity :-)

**Comment by wyf2020** [comment]
We sincerely apologize for the delay in making our training code open-source. It's now available and open for use.
